index.html

```diff
@@ -74,7 +74,7 @@
   <!-- Bootstrap Core CSS -->
   <link href="custom/css/bootstrap.min.css" rel="stylesheet" type="text/css">
   <!-- Font Awesome Icons -->
-  <link rel="stylesheet" href="./font-awesome/css/font-awesome.min.css">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">
```

Comment:
Don't do this since a lot of logos depend upon v4.7.0 of the library which is available in local. You can either include the cdn for https://fontawesome.com/v4.7.0/icons/ in case the close button is not present in the local file.